src/modules/Builder/components/ToggleItemContainer/ToggleItemContainer.types.ts

```diff
@@ -21,5 +21,6 @@ export type ToggleItemProps = {
   tooltip?: string;
   errorMessage?: string | null;
   hasError?: boolean;
+  headerToggling?: boolean;
```

Comment:
It's better to introduce the opposite property (I suppose, this feature will be applied in almost all places, it will reduce changes) for ex. `shouldSuppressHeaderToggling` and add it in places where it applicable.

At the same time, change the logic `const isHeaderClickable = !shouldSuppressHeaderToggling && !isOpenDisabled;` and use `shouldSuppressHeaderToggling = false` by default.

And could you please check if the following components should have header toggling:
- AudioPlayer.tsx
- Drawing.tsx
- ScoresAndReports.tsx
- ScoreContent.tsx
- SectionContent.tsx
- SubscalesConfiguration.tsx
- [ready] ButtonsScreen.tsx
- [ready] FixationScreen.tsx
- [ready] StimulusScreen.tsx
- [ready] BlockSequences.tsx
- [ready] Instruction.tsx
- [ready] NameDescription.tsx

Reply:
Ah sorry I think my diff is showing more than I actually changed because it’s out of date with its base, I will re-rebase to get a clearer diff